clients/python-wrapper/README.md

```diff
@@ -45,6 +45,52 @@ main_branch = repo.create(storage_namespace="<storage_namespace>").branch(branch
 ...
 ```
 
+## Examples
+
+### Print sizes of all objects in lakefs://repo/main~2
+
+```py
+ref = lakefs.Repository("repo").ref("main~2")
+for obj in ref.objects():
+  print(f"{o.path}: {o.size_bytes}")
+```
+
+### Difference between two branches
+
+```py
+for i in lakefs.Repository("repo").ref("main").diff("twig"):
+   print(i)
+```
+
+Naturally you could use ref expressions, for instance `.diff("main~2")` also
+works.
+
+### Search a stored object for a string
+
+```py
+with lakefs.Repository("repo").ref("main").object("path/to/data").reader(mode="r") as f:
+   for l in f:
+     if "quick" in l:
+	   print(l)
+```
+
+### Upload and commit some data
+
+```py
+with lakefs.Repository("golden").branch("main").object("path/to/new").writer(mode="wb") as f:
+   f.write(b"my data")
+
+# Returns a Reference
+lakefs.Repository("golden").branch("main").commit("added my data using lakeFS high-level SDK")
```

Comment:
Returns a `Commit`, no? That's not a subclass of `Reference` in Python, since those can be used in object queries.